tests/integration/test_storage.sh

```diff
@@ -35,8 +35,7 @@ setup () {
     # Perform setup routines here.
     copy /etc/sysconfig/docker-storage-setup /etc/sysconfig/docker-storage-setup.atomic-tests-backup
     TEST_DEV_1=/dev/vdb
-    findmnt -o SOURCE | grep "^$TEST_DEV_1" | uniq | xargs -r umount
-    wipefs -a "$TEST_DEV_1"
+    wipefs -a ${TEST_DEV_1}
```

Comment:
do you need -f here in case it's mounted?

Reply:
@giuseppe Can you address this?

Reply:
-f, --force
              Force erasure, even if  the  filesystem  is  mounted.   This  is
              required  in  order  to  erase  a partition-table signature on a
              block device.

Reply:
Opened a PR here: https://github.com/projectatomic/atomic/pull/1111

Reply:
Sorry, was offline today. I updated the PR below ⬇️ to just conditionalize on the `grep`. See the commit message for details.

> do you need -f here in case it's mounted?

Hmm, that sounds dangerous. We definitely want to make sure it's completely unmounted before wiping signatures, right? (So we *want* it to fail in case our `umount` bit didn't kick in somehow). The reason I originally removed the `findmnt | grep | umount` pipeline was because there is nothing to unmount in the first place in the new CI setup. But I restored it to better cover the local dev cases.